README.rst

```diff
@@ -274,6 +274,27 @@ This is how a very verbose distributed call would look like::
 The results should be show in the terminal where ``delgado`` is running, not in
 Vim.
 
+Configuration
+-------------
+Test directory
+^^^^^^^^^^^^^^
+By default the project test directory is ``tests`` (i.e. test files are assumed
+to be in ``/path/to/project/test``). The global variable ``pytest_test_dir`` may
+be used to change this, for example::
+
+    let g:pytest_test_dir = 'test_suite'
+
+configures the test directory to be ``/path/to/project/test_suite``
+
+Test runner
+^^^^^^^^^^^
+By default the test runner is ``tests.py``.The global variable
```

Comment:
I am not clear on what "test runner" is here. Initially I'd assumed it meant a different executable, but after looking at the code changes, it seems that it is meant to point to a single test file.

Can this variable and the docs references be updated to reflect that this is the test file and doesn't have anything to do with the test runner itself?